test/hotkey.test.ts

```diff
@@ -151,3 +151,64 @@ test('allows to override hotkeys', () => {
   press(window, { code: 'KeyQ', key: 'й' })
   equal(clicked, 'bqb')
 })
+
+test('dont serching in data-keyux-ignore-hotkeys', () => {
+  let window = new JSDOM().window
+  startKeyUX(window, [hotkeyKeyUX()])
+  window.document.body.innerHTML = `
+    <ul>
+      <li data-keyux-ignore-hotkeys tabindex="0">
+        Item 1
+        <button aria-keyshortcuts="v">1</button>
+      </li>
+      <li tabindex="0">
+        Item 2
+        <button aria-keyshortcuts="v">2</button>
+      </li>
+    </ul>
+  `
+
+  let result = 0
+  let buttons = window.document.querySelectorAll('button')
+  for (let button of buttons) {
+    button.addEventListener('click', () => {
+      result += parseInt(button.textContent!)
+    })
+  }
+  press(window, { key: 'v' })
+  equal(result, 2)
+
+  press(window, { code: 'KeyV', key: 'м' })
+  equal(result, 2)
+})
+
+test('ignore data-keyux-ignore-hotkeys, with data-keyux-hotkeys', () => {
```

Comment:
We need much more tests.

For instance, we need test where we move focus to `data-keyux-ignore-hotkeys`‘s element to test that it opens it’s hotkeys.

We need to test `data-keyux-hotkeys` pointing to another part of DOM like:

```html
  <li data-keyux-hotkeys="pannel">
    …
  </li>
</ul>
<div id="pannel" data-keyux-ignore-hotkeys>
  <button aria-keyshortcuts="v"></button>
</div>
```